plugins/com.google.cloud.tools.eclipse.sdk/META-INF/MANIFEST.MF

```diff
@@ -12,6 +12,8 @@ Require-Bundle: com.google.cloud.tools.appengine;bundle-version="0.4.2",
  org.eclipse.e4.core.di;bundle-version="1.5.0"
 Import-Package: com.google.common.base;version="[20.0.0,21.0.0)",
  com.google.common.collect;version="[20.0.0,21.0.0)",
+ org.eclipse.core.runtime,
```

Comment:
I don't see where this is used.

Reply:
I was tinkering with the Eclipse `Job`. I pushed a commit that sketches the use of it to report progress through the Eclipse `Job` framework.